packages/grafana-eslint-rules/rules/no-untranslated-literals.cjs

```diff
@@ -0,0 +1,46 @@
+// @ts-check
+const { ESLintUtils, AST_NODE_TYPES } = require('@typescript-eslint/utils');
+
+const createRule = ESLintUtils.RuleCreator(
+  (name) => `https://github.com/grafana/grafana/blob/main/packages/grafana-eslint-rules/README.md#${name}`
+);
+
+const noUntranslatedLiterals = createRule({
+  create(context) {
+    return {
+      JSXText(node) {
+        const ancestors = context.getAncestors();
+        const isEmpty =  !node.value.trim();
+        const hasTransAncestor = ancestors.some((ancestor) => {
```

Comment:
Is my understanding correct that this only checks for untranslated children? i.e. no string literal props are considered?

Reply:
Yes, it's in the [documentation](https://github.com/grafana/grafana/blob/e8b8b2a593a310ca68db92b8d7fdfc40dc1ee199/packages/grafana-eslint-rules/README.md?plain=1#L119) :)